bp_common/syn/Makefile.vcs

```diff
@@ -23,6 +23,13 @@ VCS_OPTIONS  = +vcs+finish+$(VCS_TIMEOUT) # Change this to run longer / shorter
 VCS_OPTIONS += +vcs+lic+wait              # Wait for license
 VCS_OPTIONS += +libext+.v+.vlib+.vh       # Find library files with these extensions
 VCS_OPTIONS += +vcs+loopdetect +vcs+loopreport
+ifeq ($(SYNTH_P), 1)
```

Comment:
Let's call this SIM_SYNTH_VIVADO_P, (to contrast with DC_SYNTH_P)

Reply:
additionally, let's make these target-specific variables i.e.

sim_vivado.v: SIM_SYNTH_VIVADO_P := 1
sim_vivado.v: VCS_OPTIONS += -y $(XILINX_VIVADO)/data/verilog/src

etc.